packages/block-library/src/details/block.json

```diff
@@ -0,0 +1,54 @@
+{
+	"$schema": "https://schemas.wp.org/trunk/block.json",
+	"apiVersion": 2,
+	"__experimental": true,
+	"name": "core/details",
+	"title": "Details",
+	"category": "text",
+	"description": "A block that displays a summary and shows or hides additional content.",
+	"keywords": [ "disclosure", "summary", "hide", "transcript" ],
+	"textdomain": "default",
+	"attributes": {
+		"showContent": {
+			"type": "boolean",
+			"default": false
+		}
+	},
+	"supports": {
+		"align": true,
+		"color": {
+			"gradients": true,
+			"link": true,
+			"__experimentalDefaultControls": {
+				"background": true,
+				"text": true,
+				"link": true
+			}
+		},
+		"__experimentalBorder": {
+			"radius": true,
+			"color": true,
+			"width": true,
+			"style": true
+		},
```

Comment:
There's a small disparity between the editor and frontend when using border radius:
## Editor
![Screen Shot 2023-04-05 at 4 47 49 pm](https://user-images.githubusercontent.com/677833/230030927-9f0b0b23-c7e0-452b-8dcc-2df255f6c2a3.png)

## Frontend
![Screen Shot 2023-04-05 at 4 47 41 pm](https://user-images.githubusercontent.com/677833/230030975-53532c2e-0811-4d88-86cf-f700b7ecff18.png)

It's something that can be fixed in a follow-up and shouldn't block merging.

Reply:
I am trying to reproduce this.
I am using Twenty Twenty-Three.
- I added a border with border-radius on the Details.
-  Padding, preset 1 , on the summary and content.
- A background color on the summary.

I do not see the clipping on macOS, in Firefox, Safari, or Chrome

Reply:
This seems to work now too:
![Screen Shot 2023-04-11 at 11 14 21 am](https://user-images.githubusercontent.com/677833/231046893-bb358dea-cd52-4eea-b2e4-6aad5345297c.png)

Very strange. I tried switching between a few different themes and couldn't reproduce.

Maybe it was an issue with my local environment.

Reply:
that is a relief, thank you for double checking